.idea/misc_diff.java

```diff
@@ -0,0 +1,1374 @@
+//                         JSONObject product = jSONArray.getJSONObject(i);
+//                         String title = product.getString("title");
+//                         String price = product.getString("price");
+//                         String website = product.getString("website");
+//                         if(!title.isEmpty()){
+//                             rows.add("Name: "+title+ "\nPrice: "+price+ "\nWebsite: "+website);
+//                         }
+//                     }
+//                 }
+//             } catch (Exception e) {
+//                 e.printStackTrace();
+//             }
+
+//             fetchHandler.post(new Runnable() {
+//                 @Override
+//                 public void run() {
+//                     if(progressDialog.isShowing())
+//                         progressDialog.dismiss();
+//                     adapter.notifyDataSetChanged();
+//                 }
+//             });
+//         }
+//     }
+
+//     static Toast t;
+//     private static void makeToast(String s) {
+//         if (t != null) t.cancel();
+//         t = Toast.makeText(context, s, Toast.LENGTH_SHORT);
+//         t.show();
+//     }
+// }
+// package com.xa.xpensauditor;
+
+// import androidx.annotation.NonNull;
+// import androidx.appcompat.app.AppCompatActivity;
+// import androidx.recyclerview.widget.RecyclerView;
+
+// import android.annotation.SuppressLint;
+// import android.os.Bundle;
+// import android.widget.Button;
+// import android.widget.DatePicker;
+
+// import com.anychart.AnyChart;
+// import com.anychart.AnyChartView;
+// import com.anychart.chart.common.dataentry.DataEntry;
+// import com.anychart.chart.common.dataentry.ValueDataEntry;
+// import com.anychart.charts.Cartesian;
+// import com.anychart.core.cartesian.series.Column;
+// import com.anychart.enums.Anchor;
+// import com.anychart.enums.HoverMode;
+// import com.anychart.enums.Position;
+// import com.anychart.enums.TooltipPositionMode;
+// import com.firebase.client.Firebase;
+// import com.google.firebase.auth.FirebaseAuth;
+// import com.google.firebase.database.DataSnapshot;
+// import com.google.firebase.database.DatabaseError;
+// import com.google.firebase.database.DatabaseReference;
+// import com.google.firebase.database.FirebaseDatabase;
+// import com.google.firebase.database.ValueEventListener;
+
+// import java.util.ArrayList;
+// import java.util.Calendar;
+// import java.util.Collections;
+// import java.util.Date;
+// import java.util.GregorianCalendar;
+// import java.util.HashMap;
+// import java.util.List;
+// import java.util.Map;
+// import java.util.TreeMap;
+
+// public class MonthWiseVis extends AppCompatActivity {
+//     private Firebase mRootRef;
+//     private Firebase RefUid,RefTran,RefUid1;
+//     private Firebase RefName,RefEmail,RefPhnnum;
+//     private Firebase RefCat,RefFood,RefHealth,RefTravel,RefEdu,RefBills,RefHomeNeeds,RefOthers,RefUncat;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).